dashboard/app/models/gallery_activity.rb

```diff
@@ -14,10 +13,8 @@
 #
 # Indexes
 #
-#  index_gallery_activities_on_activity_id                  (activity_id)
 #  index_gallery_activities_on_app_and_autosaved            (app,autosaved)
 #  index_gallery_activities_on_level_source_id              (level_source_id)
-#  index_gallery_activities_on_user_id_and_activity_id      (user_id,activity_id)
```

Comment:
Note that once the activity_id subindex is removed, the entire index becomes redundant with :index_gallery_activities_on_user_id_and_level_source_id.